CONTRIBUTING.md

```diff
@@ -84,6 +84,7 @@ below:
  - Mark Worsfold (Met Office, UK)
  - Bruce Wright (Met Office, UK)
  - Ying Zhao (Bureau of Meteorology, Australia)
+ - Anzer Khan (Met Office, UK)
```

Comment:
This section is sorted alphabetically on surname, so please move your name to the correct position.